spec/javascripts/packs/src/apps/mydb/elements/details/researchPlans/ResearchPlanDetails.spec.js

```diff
@@ -1,62 +1,66 @@
-import React from "react";
-import expect from "expect";
-import Enzyme, { shallow } from "enzyme";
-import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
-
-import ResearchPlanFactory from "factories/ResearchPlanFactory";
-import AttachmentFactory from "factories/AttachmentFactory";
-import uuid from "uuid";
+/* global describe, context, it */
+
+import React from 'react';
+import expect from 'expect';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+
+import ResearchPlanFactory from 'factories/ResearchPlanFactory';
+import AttachmentFactory from 'factories/AttachmentFactory';
+import uuid from 'uuid';
 // although not used import of ElementStore needed to initialize
 // the ResearchPlanDetails.js component. It must be executed before
 // the import of ResearchPlanDetails. Beware of the linter which
 // may put it at the end of the imports !!!
 
 // eslint-disable-next-line no-unused-vars
-import ElementStore from "src/stores/alt/stores/ElementStore";
+import ElementStore from 'src/stores/alt/stores/ElementStore';
 
-import ResearchPlanDetails from "src/apps/mydb/elements/details/researchPlans/ResearchPlanDetails";
+import ResearchPlanDetails from 'src/apps/mydb/elements/details/researchPlans/ResearchPlanDetails';
 
 Enzyme.configure({
   adapter: new Adapter(),
 });
 
-describe("ResearchPlanDetails", async () => {
-  const FIELD_ID_IMAGE = "entry-001";
-  const FIELD_ID_NO_IMAGE = "entry-002";
-  describe(".handleBodyChange", async () => {
-    context("on non existing field", async () => {
-      it(" expecting nothing was changed", async () => {
+describe('ResearchPlanDetails', async () => {
+  const FIELD_ID_IMAGE = 'entry-001';
+  const FIELD_ID_NO_IMAGE = 'entry-002';
+  describe('.handleBodyChange', async () => {
+    context('on non existing field', async () => {
+      it(' expecting nothing was changed', async () => {
         const researchPlanWithImage = await ResearchPlanFactory.build(
-          "with_image_body_field"
+          'with_image_body_field'
         );
         const wrapper = shallow(
           <ResearchPlanDetails
             researchPlan={researchPlanWithImage}
             toggleFullScreen={() => {}}
-          />
+          />,
+          { disableLifecycleMethods: true }
```

Comment:
why not stub the `CommentActions.fetchComments(researchPlan)` ?

Reply:
resolved